### DISCUSSION TITLE
Output OCR layer onto a different file

### DISCUSSION THREAD

**Discussion by noahjutz** [ocrmypdf/OCRmyPDF#1408] [Q&A]
Is it possible to use file A as tesseract input in order to get the invisible OCR layer, and then paste that OCR layer onto a different file B? In other words, I want the OCR layer of file A and the background layer of file B.

Motivation: File A is very high quality, but file B is very compressed and will not yield the same OCR results. `--optimize=3` is not sufficient for my use case.

**Comment by jbarlow83** [comment]
You could use qpdf's overlay-underlay features (see `--help=overlay-underlay`) to overlay the high quality file (no OCR) over the low quality one (with OCR). The low quality images would be present but invisible.

OCRmyPDF can't do this out of the box. It has machinery to do similar things, but qpdf from command line would be easier for your use case.
